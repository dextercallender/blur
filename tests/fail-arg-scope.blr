int scope() {
    int x = 5;
    return x;
}

void main() {
    scope();
    println(x);
}
