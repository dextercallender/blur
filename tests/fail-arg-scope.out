Fatal error: exception Exceptions.UnknownVariable("x")
